tests/Doctrine/Tests/Common/Annotations/DocParserTest.php

```diff
@@ -1398,6 +1398,14 @@ public function testWillParseAnnotationSucceededByANonImmediateDash()
         self::assertCount(1, $result);
         self::assertInstanceOf(SomeAnnotationClassNameWithoutConstructorAndProperties::class, $result[0]);
     }
+
+    public function testMultiByteAnnotationLocaleTR()
+    {
+        setlocale(LC_ALL, 'tr_TR.utf8', 'tr_TR');
```

Comment:
Are we resetting the locale after this test?

Reply:
Add reset locale after this test